test/e2e_new/tracing_test.go

```diff
@@ -48,15 +55,16 @@ func TestTracingHeaders(t *testing.T) {
 		knative.WithLoggingConfig,
 		knative.WithTracingConfig,
 		k8s.WithEventListener,
+		tracing.WithZipkin,
 		environment.Managed(t),
 	)
 
-	env.Test(ctx, t, TracingHeadersUsingOrderedDelivery())
+	env.Test(ctx, t, TracingHeadersUsingOrderedDeliveryWithTraceExported())
```

Comment:
would it be possible to do this upstream?

Reply:
Sure. I will send it upstream. I'm just testing it here.